installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -167,13 +167,13 @@
             "SERVICE_HOSTNAME": "{SVC_HOST}"
           },
           "STG": {
-            "MGMT_ENV": "stg",
+            "MGMT_ENV": "prod",
             "MGMT_ORG": "{MGMT_ORG}",
             "MGMT_HOST": "{MGMT_HOST}",
             "SERVICE_HOSTNAME": "{SVC_HOST}"
           },
           "DEV": {
-            "MGMT_ENV": "dev",
+            "MGMT_ENV": "prod",
```

Comment:
I think this has to be reverted. `dev` should point to it's own  dedicated env (`dev`) in apigee too.

Reply:
Done